modules/statics/src/coins.ts

```diff
@@ -4089,6 +4089,14 @@ export const coins = CoinMap.fromCoins([
     '0xeb40045347531f06179ffb3c6be33d5b35a52ebd',
     UnderlyingAsset.GHUB
   ),
+  erc20(
+    'd3c6a9b2-1551-401c-b576-db429d48b895',
+    'ghst',
+    'Aavegotchi',
+    18,
+    '0x3F382DbD960E3a9bbCeaE22651E88158d2791550',
```

Comment:
please make sure all letters are in lowercase

Reply:
done